package.json

```diff
@@ -29,12 +31,11 @@
   },
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-core": "^6.7.2",
     "babel-preset-es2015": "^6.6.0",
+    "babel-register": "^6.7.2",
     "chai": "^3.5.0",
     "eslint": "2.3.0",
     "eslint-config-airbnb": "6.1.0",
-    "eslint-plugin-react": "^4.2.3",
```

Comment:
It's unnecessary when using [AirBnB's base config](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnbbase).